api/test/wfl/integration/source_test.clj

```diff
@@ -192,10 +193,10 @@
                 (is (= expected-num-records (count records))))
               (testing "all snapshot jobs were updated and corresponding snapshot ids were inserted"
                 (is (every? record-updated? records))))))
-        (let [stopped-source (source/update-source!
-                              (jdbc/with-db-transaction [tx (postgres/wfl-db-config)]
-                                (source/stop-source! tx source)
-                                (reload-source tx source)))]
+        (let [stopped-source (:source (source/update-source!
```

Comment:
I think this gets simpler too with this advice:

https://github.com/broadinstitute/wfl/pull/551/files#r777601418